workflow_test.go

```diff
@@ -0,0 +1,80 @@
+package workflow_test
+
+import (
+	"context"
+	"flag"
+	"testing"
+
+	"github.com/codingpop/workflow-trigger"
+)
+
+var owner = flag.String("owner", "", "Repository owner")
+var repo = flag.String("repo", "", "GitHub repository name")
+var accessToken = flag.String("access-token", "", "Personal access token")
+var eventType = flag.String("event-type", "", "A custom webhook event name")
+
+func TestTrigger(t *testing.T) {
+	p := workflow.Params{
```

Comment:
If you allow the base URL to be configured, you could encapsulate this test to run against a local HTTP server, instead of pinging the github API every time you run the tests.